comm/src/main/scala/coop/rchain/comm/transport/TcpTransportLayer.scala

```diff
@@ -208,34 +208,29 @@ class TcpTransportLayer(
   def broadcast(peers: Seq[PeerNode], msg: Protocol): Task[Seq[CommErr[Unit]]] =
     innerBroadcast(peers, msg)
 
-  def handleToStream(toStream: ToStream): Task[Unit] = {
+  private def streamToPeer(peer: PeerNode, path: Path, sender: PeerNode): Task[Unit] = {
```

Comment:
name is misleading, `handle` convention was always good, you knew you are now "handling" a message, you are "on the server side"

Reply:
I don't agree. We are not handling a message (`ToStream') here anymore. It's a function with parameters. We are on the client side and want to stream a file to some peer node (server side). Maybe I can change the parameter name `path` to `file`?